graphkb/constants.py

```diff
@@ -172,3 +172,45 @@ def __getitem__(self, key):
     "mutations": "mut",
     "nonsense": ">",
 }
+
+# For match.type_screening() [KBDEV-1056]
+DEFAULT_NON_STRUCTURAL_VARIANT_TYPE = 'mutation'
+STRUCTURAL_VARIANT_SIZE_THRESHOLD = 50  # bp
+STRUCTURAL_VARIANT_TYPES = [
+    "structural variant",
+    "insertion",
+    "in-frame insertion",
+    "deletion",
+    "deletion polymorphism",
+    "in-frame deletion",
+    "translocation",
+    "inverted translocation",
+    "inversion",
+    "indel",
+    "fusion",
+    "out-of-frame fusion",
+    "oncogenic fusion",
+    "in-frame fusion",
+    "disruptive fusion",
+    "duplication",
+    "internal duplication",
+    "tandem duplication",
+    "internal tandem duplication",
+    "itd",
+    "domain duplication",
+    "kinase domain duplication",
+    "copy variant",
+    "copy number variation",
+    "copy number variant",
+    "copy loss",
+    "copy number loss",
+    "shallow deletion",
+    "deep deletion",
+    "gene deletion",
+    "copy gain",
+    "copy number gain",
+    "low level copy gain",
+    "amplification",
+    "focal amplification",
```

Comment:
Pretty sure all the copy change effects are considered something separate.

Structural variants involve changes within a gene somehow.

Reply:
Interesting.
Here is the complete Vocabulary graph in regard to structural variants:

![structural_variants](https://github.com/bcgsc/pori_graphkb_python/assets/4022327/7bb7b864-d8fd-407a-a095-e69cd674ebe7)

If  they don't belong there, we'll have to make important changes to our variant type classification.

I thought that copy gain/loss or amplification could be the result of gene/chromosome being deleted or duplicated somehow. Again, do you know who could make the call on that one? I'll ping Erin on the ticket.

Reply:
Okay - again we'll go with this for now and ask clinical group for feedback.

I can say that the gsc_report would never be able to call a copy loss or amplification a 'Structural Variant'.
There would not be things like exons where the change occurred, etc.